src/chunker.c

```diff
@@ -8,13 +8,70 @@
  * optimization. See RFC1950 for details. */
 #define BYTES_PUSH_MAX 5552U
 
+/* Unless otherwise specified, take this as the default average chunk size */
+#define DEFAULT_CHUNK_SIZE_AVG (16 * 1024)
```

Comment:
shouldn't that have the "U" suffix? after all we might end up compare against that, and gcc might cry if we compare against unsigned...